src/entries/background/handlers/handleProviderRequest.ts

```diff
@@ -364,7 +363,7 @@ export const handleProviderRequest = ({
         extensionUrl,
         host,
       });
-      logger.error(new RainbowError('Chain Id not supported'), {
+      console.warn('Chain Id not supported', {
```

Comment:
Will this warn bubble up to the inpage script? Probably can log as info otherwise if user can't see it